{{cookiecutter.project_slug}}/fabfile.py

```diff
@@ -497,6 +543,38 @@ def import_db(c, dump_file=None, with_media=False):
         import_media(c)
 
 
+def jira_release(c, command, **kwargs):
+    full_kwargs = dict()
+
+    if c.config["environment"] == "prod":
```

Comment:
Move this condition into `comment_and_close_on_jira` ? Otherwise it always overrides the `command` argument